arcaflow_plugin_coremark_pro/coremark_pro_plugin.py

```diff
@@ -146,13 +146,15 @@ def certify_all(
                 "SingleCore": float(line_list[2]),
                 "Scaling": float(line_list[3]),
             }
-            # Collect the per-benchmark iterations from the log file
-            if line_name != "CoreMark-PRO":
-                with open(file=run_log_path, encoding="utf-8") as file:
-                    for line in file:
-                        line_list = line.split()
-                        if "median single" in line and line_list[2] == line_name:
-                            ca_results[line_name]["Iterations"] = int(line_list[7])
+
+    # Collect the per-benchmark iterations from the log file
+    with open(file=run_log_path, encoding="utf-8") as file:
+        for line in file:
+            if "median single" not in line or "CoreMark-PRO" in line:
+                continue
+            line_list = line.split()
+            if line_list[2] in ca_results:
+                ca_results[line_list[2]]["Iterations"] = int(line_list[7])
```

Comment:
I think that the old `line_name` local made this a little easier to read, compared to the nested key `ca_results[line_list[2]]`. Not a big deal since it's two adjacent lines, but `benchmark = line_list[2] ; if benchmark in ca_results` makes this just a little easier to read. No big deal: I'll approve and pre-resolve the comment.

Reply:
I kinda meant to add that after testing the change, and I forgot.